src/steps/mdx-to-gridtables.ts

```diff
@@ -141,9 +127,11 @@ export default function mdxToBlocks(ctx: Helix.UniversalContext) {
       });
     }
 
-    const rowsToInsert = listToMatrix(slotsToInsert, slots.length);
+    // Calculate rows to insert based on total slots
+    const rowsToInsert = isHorizontalLine ? [] : listToMatrix(slotsToInsert, slots.length);
```

Comment:
```suggestion
    const rowsToInsert = isHorizontalLine ? [][] : listToMatrix(slotsToInsert, slots.length);
```

listToMatrix returns a 2D array: https://github.com/aemsites/devsite-runtime-connector/blob/12210503c066f8d59633298323e2b3741531cbba/src/steps/mdx-to-gridtables.ts#L47

Reply:
@BaskarMitrah sounds good! Once you do that change, please check if we still need lines 133-134 and 162-169. If not needed anymore, please revert those lines

Reply:
Actually sorry, you might wanna revert back to just
```suggestion
    const rowsToInsert = listToMatrix(slotsToInsert, slots.length);
```

because slots is empty. so Ideally this should return [][] for isHorizontalLine. Please test and confirm first.